config/cloudpaks/cp4a/resources/templates/0201-icp4a-cluster.yaml

```diff
@@ -11,27 +11,29 @@ metadata:
     argocd.argoproj.io/sync-wave: "201"
   namespace: {{.Values.argocd_app_namespace}}
 spec:
-  appVersion: 22.0.1
+  appVersion: 22.0.2
```

Comment:
`appVersion` was the primary change to the resource.

Bulk of the changes was just ordering fields around to match the order of the YAML file in the form editor.